framework/src/main/java/org/checkerframework/framework/qual/ConditionalPostconditionAnnotation.java

```diff
@@ -13,9 +13,10 @@
  *
  * <p>The annotation E that is meta-annotated as {@link ConditionalPostconditionAnnotation} must
  * have an element called {@code expression} that is an array of {@code String}s of the same format
```

Comment:
This is not true.
`EnsuresLTLengthOfIf.List` is meta-annotated with `@ConditionalPostconditionAnnotation`, but it does not have an element called `expression`.

Reply:
@mernst Please read the whole paragraph of the documentation

Reply:
> @mernst Please read the whole paragraph of the documentation

I did.  By the way, it is rude to write a short response that says I didn't do my work, but that does not respond to my comment and that does nothing to help resolve the misunderstanding.

Let's read this together,  The sentence is:
"The annotation E ... must have an element called `expression` that is an array of Strings ... or a list of conditional postcondition annotations ...".
In other words, the sentence says that there must be an element called `expression`, which might have one of two different types.  That seems to be in conflict with its use in the code.

Am I reading the sentence wrong, reading the code wrong, or have I made some other mistake?  Please clarify.

Reply:
> I did. By the way, it is rude to write a short response that says I didn't do my work, but that does not respond to my comment and that does nothing to help resolve the misunderstanding.

@mernst Sorry If you felt like that. That was not the intention.
> Let's read this together, The sentence is:
> "The annotation E ... must have an element called `expression` that is an array of Strings ... or a list of conditional postcondition annotations ...".
> In other words, the sentence says that there must be an element called `expression`, which might have one of two different types. That seems to be in conflict with its use in the code.

I meant to say that after before and after `or` they are both different sentences irrespective of each other. I think you are saying to write the documentation like this-
```
The annotation E that is meta-annotated as {@link ConditionalPostconditionAnnotation} must have an element called {@code expression} that is an array of {@code String}s of the same format and with the same meaning as the value {@code expression} in {@link EnsuresQualifierIf} or it must have a list of conditional postcondition annotations in an array{@code value} and with the same meaning as the {@code value} in {@link EnsuresQualifiersIf}. E must also have an element {@code result} with the same meaning as the element {@code result} in {@link EnsuresQualifierIf}.
```
Is this correct?

Reply:
That is better -- at least it is not wrong as the previous version was.